### PR TITLE
Skill style rework

### PR DESCRIPTION
**Pull request by guswadsworth12**
Reworked a lot of the styling to be more compact

### CODE REVIEW

File: templates/actor/parts/actor-common.hbs [hunk 1]
```diff
@@ -1,25 +1,41 @@
...
-  <span>{{localize 'E20.initiative'}}</span>
-  <select name="system.initiative">
-    {{selectOptions @root.config.initiativeShifts selected=system.initiative localize=true}}
-  </select>
 </div>
-
-{{!-- Essences/Skills --}}
-{{> "systems/essence20/templates/actor/parts/actor-character.hbs"}}
-<h3 class="skills-label">{{localize 'E20.skills'}}</h3>
+</div>
```
Comment (phildominguez) [inline templates/actor/parts/actor-common.hbs:35]: I think this is an extra `</div>`, then indent lines 2-33.

File: templates/actor/parts/actor-common.hbs [hunk 2]
```diff
@@ -1,25 +1,41 @@
...
-  <div class="resource flex-group-center">
-    <label for="system.health.value" class="resource-label">{{localize 'E20.health'}}</label>
-    <div class="resource-content flexrow flex-center flex-between">
-      <input type="text" name="system.health.value" value="{{system.health.value}}" data-dtype="Number" />
-      <span> / </span>
-      <input type="text" name="system.health.max" value="{{system.health.max}}" data-dtype="Number" />
+<div class="common-container">
+<div class="common-container-1">
+{{!-- Essences/Skills --}}
+{{> "systems/essence20/templates/actor/parts/actor-character.hbs"}}
```
Comment (phildominguez) [inline templates/actor/parts/actor-common.hbs:4]: Indent here

**Review by phildominguez — CHANGES_REQUESTED** [review]
Looks good! Some tiny cleanup.

File: templates/actor/parts/actor-common.hbs (hunk 1 above)
Comment (guswadsworth12) [reply templates/actor/parts/actor-common.hbs]: This is actually not. But i applied the changes to the branch

File: templates/actor/parts/actor-common.hbs (hunk 2 above)
Comment (guswadsworth12) [reply templates/actor/parts/actor-common.hbs]: Done

File: templates/actor/actor-npc-sheet.hbs [hunk 3]
```diff
@@ -16,9 +16,9 @@
   <section class="sheet-body">
     {{!-- Skills Tab --}}
     <div class="tab skills" data-group="primary" data-tab="skills">
-      {{> "systems/essence20/templates/actor/parts/actor-common.hbs"}}
-      {{> "systems/essence20/templates/actor/parts/actor-character.hbs"}}
-      {{> "systems/essence20/templates/actor/parts/actor-traits.hbs"}}
+      {{> "systems/essence20/templates/actor/parts/actor-common.hbs"}} 
+    {{!--   {{> "systems/essence20/templates/actor/parts/actor-character.hbs"}}
+      {{> "systems/essence20/templates/actor/parts/actor-traits.hbs"}}--}}
```
Comment (phildominguez) [inline templates/actor/actor-npc-sheet.hbs:21]:
```suggestion
```
Might as well just remove

**Review by phildominguez — APPROVED** [review]
👍

**Pull request merged**